kafka-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchKafkaDAOV5.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.netflix.conductor.dao.es5.index;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.conductor.annotations.Trace;
+import com.netflix.conductor.common.metadata.events.EventExecution;
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.TaskExecLog;
+import com.netflix.conductor.common.run.TaskSummary;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.common.run.WorkflowSummary;
+import com.netflix.conductor.core.events.queue.Message;
+import com.netflix.conductor.dao.ProducerDAO;
+import com.netflix.conductor.dao.kafka.index.utils.RecordTypeConstants;
+import com.netflix.conductor.elasticsearch.ElasticSearchConfiguration;
+import org.elasticsearch.client.Client;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Manan
+ */
+@Trace
+@Singleton
+public class ElasticSearchKafkaDAOV5 extends ElasticSearchDAOV5 {
```

Comment:
Hi @apanicker-nflx This class extends ElasticSearchDAOV5 so that all read operations will route to ElasticSearchDAOV5. But overrides all index operations which publish to kafka topic.